src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -524,7 +524,7 @@ export class EditorService extends Disposable implements EditorServiceImpl {
 				id: DEFAULT_EDITOR_ASSOCIATION.id,
 				label: DEFAULT_EDITOR_ASSOCIATION.displayName,
 				detail: DEFAULT_EDITOR_ASSOCIATION.providerDisplayName,
-				describes: (currentEditor) => currentEditor.matches(this.activeEditor),
+				describes: (currentEditor) => this.fileEditorInputFactory.isFileEditorInput(currentEditor) && currentEditor.matches(this.activeEditor),
```

Comment:
What is `describes` supposed to check? I thought it was checking the _type_ of the given editor input, but this is checking that the input is a particular instance which is the active editor?

Reply:
`describes` is used to figure out if your contribution is the editor passed into the function. It's normally the active editor so in this case we see if it's a fileEditor and if so make sure it's not the active one. I used to have the function be called `active`, but Ben felt it was better as describes.

Reply:
You could probably get away without the matches, but I don't want to break it this close to release. So I just added the important thing which is to make sure it's a file editor input.

Reply:
I see, yeah it seems like you would just want the file editor check